File: Fibonacci.py
'''Fibonacci Sequence - Enter a number and have the program generate the Fibonacci sequence to that number or to the Nth number.'''
def fibonaci(nr):
    if nr == 1 or nr == 2:
        return 1
    else:
        return fibonaci(nr - 1) + fibonaci(nr - 2)


user_input = int(input("What element from the Fibonacci Sequence you want ?"))
print(fibonaci(user_input))

File: Fibonacci.cpp
/*
Fibonacci Sequence - Enter a number and have the program generate the Fibonacci sequence to the Nth number.
*/
#include<iostream>
using namespace std;
int fibonaci(int nr)
{
	if (nr == 1 || nr == 2)
	{
		return 1;
	}
	else
	{
		return fibonaci(nr - 1) + fibonaci(nr - 2);
	}
}
int main()
{
	int number;
	cout << "What element from the Fibonacci Sequence you want ?";
	cin >> number;
	cout << fibonaci(number);
}
